### ISSUE TITLE
Bad instructions

### ISSUE THREAD

**Issue by randygrolemund** [fireice-uk/xmr-stak#1743]
The compile instructions have numerous issues.

PS D:\git\xmr-stak> set CMAKE_PREFIX_PATH=C:\xmr-stak-dep\hwloc;C:\xmr-stak-dep\libmicrohttpd;C:\xmr-stak-dep\openssl
C:\xmr-stak-dep\libmicrohttpd : The term 'C:\xmr-stak-dep\libmicrohttpd' is not recognized as the name of a cmdlet,
function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the
path is correct and try again.
At line:1 char:45
+ ... EFIX_PATH=C:\xmr-stak-dep\hwloc;C:\xmr-stak-dep\libmicrohttpd;C:\xmr- ...
+                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\xmr-stak-dep\libmicrohttpd:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

C:\xmr-stak-dep\openssl : The term 'C:\xmr-stak-dep\openssl' is not recognized as the name of a cmdlet, function,
script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is
correct and try again.
At line:1 char:75
+ ... -stak-dep\hwloc;C:\xmr-stak-dep\libmicrohttpd;C:\xmr-stak-dep\openssl
+                                                   ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\xmr-stak-dep\openssl:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by randygrolemund** [comment]
PS D:\git\xmr-stak> "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x64
 -vcvars_ver=14.11
At line:1 char:98
+ ... l Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x64 -vcvars ...
+                                                               ~~~
Unexpected token 'x64' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken

**Comment by randygrolemund** [comment]
PS D:\git\xmr-stak> cmake --build . --config Release --target install
Error: could not load cache

**Comment by Spudz76** [comment]
Use `cmd.exe` not powershell and it works fine, I think?

Some things have different dialect such as semicolons break commands apart more like UNIX, whereas `cmd.exe` just takes all that garbage after `=` as a string without caring about the semicolons.  So wrap the right half of the set thing in quotes and it might like it better.

I don't really know powershell (other than it's annoyingly incompatible with cmd.exe) or I'd be more help.

But I doubt anything with CMake will work (your other posts) unless you use `cmd.exe` as your default shell there as well..

I don't think normal Windows installs have powershell as default?  Mainly because it's incompatible with things; exactly this issue.

**Comment by Spudz76** [comment]
ahh, do not build in the source directory

make subfolder like `build` then cd there and run commands

do `git status` and delete any crap listed as untracked.  or start over from fresh checkout haha

**Comment by randygrolemund** [comment]
Yup.... it's working now. I do not have much C experience. Thanks for your help.... really appreciate it. You rule!

**Comment by Spudz76** [comment]
I hadn't noticed the path even though PS has it in the prompt.

My eyes just skip to the > or % or # or whatever char.  Lucky I saw the PS I guess, but the errors look like some "powershell spew" that I'd seen when I tried using it once.
